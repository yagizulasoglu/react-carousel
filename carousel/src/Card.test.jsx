import { describe, it, expect } from "vitest";
import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon.js";


it("renders without crashing", function () {
  render(<Card caption={TEST_IMAGES[0].caption} src={TEST_IMAGES[0].src} currNum="0" totalNum="0" />);
});

it("matches snapshot", function () {
  const { container } = render(<Card caption={TEST_IMAGES[0].caption} src={TEST_IMAGES[0].src} currNum="0" totalNum="0" />);
  expect(container).toMatchSnapshot();
});