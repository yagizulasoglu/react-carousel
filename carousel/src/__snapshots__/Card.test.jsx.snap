// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      testing image 1
    </h4>
    <img
      alt="testing image 1"
      class="Card-image"
      src="test1.com"
    />
    <small
      class="Card-small"
    >
      Image 
      0
       of 
      0
      .
    </small>
  </div>
</div>
`;
